ja/lessons/advanced/otp-supervisors.md

```diff
@@ -1,95 +1,163 @@
 ---
-version: 1.0.1
-title: OTP スーパーバイザ
+version: 1.1.1
+title: OTP スーパバイザ
```

Comment:
Why do you want to change this?

Reply:
The current document has some fluctuation to use loan-words e.g. スーパーバイザー and スーパーバイザ for 'supervisor'. I think we should use some rule for avoiding such variation. I found a standard and a guideline listed below.
[外来語(カタカナ)表記ガイドライン](https://www.jtca.org/standardization/katakana_guide_3_20171222.pdf)
[JIS Z8301:2008](https://kikakurui.com/z8/Z8301-2011-01.html)
[JIS Z8301:2011](https://kikakurui.com/z8/Z8301-2011-02.html)
In this case, I use JIS Z8301 because it seems sound for me.

Reply:
Since your pull request appears to be the change to the already translated contents, I suppose you have to put the number of patch versions ahead, not minor.